package.json

```diff
@@ -28,24 +28,24 @@
   },
   "homepage": "https://github.com/angular/in-memory-web-api#readme",
   "peerDependencies": {
-    "@angular/common": "^6.0.0-rc.0",
-    "@angular/core": "^6.0.0-rc.0",
-    "@angular/http": "^6.0.0-rc.0",
-    "rxjs": "^6.0.0-beta.1"
+    "@angular/common": "^6.0.0",
+    "@angular/core": "^6.0.0",
+    "@angular/http": "^6.0.0",
+    "rxjs": "^6.1.0"
```

Comment:
this is a breaking change because all the client apps would be forced to update to rxjs 6.1.0 - please change this to ^6.0.0